src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/ui-data/lambda-metrics.mdx

```diff
@@ -0,0 +1,385 @@
+---
+title: Available Lamda metrics
+metaDescription: Available Lambda metrics in New Relic.
+freshnessValidatedDate: never
+
+---
+The following table lists the available Lambda metrics for your Lambda functions in New Relic:
+
+<table>
+    <thead>
+        <tr>
+            <th style={{ width: "200px" }}>
+            **Metric type**
+            </th>
+            <th>
+            **Metrics**
```

Comment:
@cbehera-newrelic I think you could use the singular form `Metric` here. Or, if you decide to insert the actual names of the metrics you'd see in the database, you could insert `Metric name`.